activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -252,6 +251,17 @@ def load_seed
         end
       end
 
+      def self.run_cmd(cmd, args, action)
+        fail run_cmd_error(cmd, args, action) unless Kernel.system(cmd, *args)
+      end
+
+      def run_cmd_error(cmd, args, action)
+        msg = "Error #{action} database: failed to execute "
+        msg << "`#{cmd} #{args.join(' ')}`. Please "
+        msg << "check that #{cmd} is present on this system, is in your PATH"
+        msg << ', and has proper permissions.'
```

Comment:
Can we add some line breaks to the message. It currently looks like that:

```
$ bin/rake db:structure:dump
Error dumping database: failed to execute `pg_dump -i -s -x -O -f /Users/senny/Projects/playground/ruby/rails/rails-master/db/structure.sql test123_development`. Please check that pg_dump is present on this system, is in your PATH, and has proper permissions.
/Users/senny/Projects/rails/activerecord/lib/active_record/tasks/database_tasks.rb:256:in `run_cmd'
/Users/senny/Projects/rails/activerecord/lib/active_record/tasks/postgresql_database_tasks.rb:64:in `structure_dump'
/Users/senny/Projects/rails/activerecord/lib/active_record/tasks/database_tasks.rb:182:in `structure_dump'
/Users/senny/Projects/rails/activerecord/lib/active_record/railties/databases.rake:273:in `block (3 levels) in <top (required)>'
Tasks: TOP => db:structure:dump
(See full trace by running task with --trace)
```

Something along the lines of:

```
$ bin/rake db:structure:dump
rake aborted!
failed to execute:
pg_dump -i -s -x -O -f /Users/senny/Projects/playground/ruby/rails/rails-master/db/structure.sql test123_development

Please check that `pg_dump` is:
 - present on this system
 - in your PATH
 - has proper permissions.

/Users/senny/Projects/rails/activerecord/lib/active_record/tasks/database_tasks.rb:256:in `run_cmd'
/Users/senny/Projects/rails/activerecord/lib/active_record/tasks/postgresql_database_tasks.rb:64:in `structure_dump'
/Users/senny/Projects/rails/activerecord/lib/active_record/tasks/database_tasks.rb:182:in `structure_dump'
/Users/senny/Projects/rails/activerecord/lib/active_record/railties/databases.rake:273:in `block (3 levels) in <top (required)>'
Tasks: TOP => db:structure:dump
(See full trace by running task with --trace)
```